index.js

```diff
@@ -34,6 +36,26 @@ module.exports = (version) => {
       }
       return ChatMessage.fromNotch(name)
     }
+
+    getDroppedItem () {
+      if (this.name !== 'item' && this.name !== 'Item' && this.name !== 'item_stack') {
+        return null // not a dropped item
+      }
+      let metadataIx
+      if (mcData.isNewerOrEqualTo('1.17')) {
```

Comment:
might be time we put features.json in minecraft-data
this kind of if version is so bad..